local_test/run_worker.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ex
+
+mkdir -m700 -p /root/.ssh
+cp local_test/id_rsa.rrrspec /root/.ssh/id_rsa
```

Comment:
id_rsa.rrrspec in the repository is 0600 and `cp` keeps the file mode when it creates a new file.